tox.ini

```diff
@@ -21,7 +21,7 @@ deps =
 	djangomaster: https://github.com/django/django/archive/master.tar.gz
     apns2>=0.3.0
```

Comment:
Looks like we have to limit the apns2 version here too

Reply:
Odd, that's not needed since it's already in `install_requires` and tox will build a sdist and install the package in a venv anyway.